asconfig.json

```diff
@@ -57,7 +57,7 @@
             },
             {
                 "name": "R3::VERSION",
-                "value": "\"1.2.0\""
+                "value": "\"9.9.9\""
```

Comment:
Why is R3 version suddenly 9.9.9?

Reply:
Development only, release asconfig is fine. See previous comment.